eox_nelp/user_profile/api/v1/views.py

```diff
@@ -56,14 +57,17 @@ def update_user_data(request):
             # This extra code block is required since the method update_account_settings just
             # allows to update fields defined in the AccountUserSerializer and the AccountLegacyProfileSerializer
             # so some fields like first_name and last_name are not editable in the standad implementation.
-            extra_account_user_fields = getattr(settings, "EXTRA_ACCOUNT_USER_FIELDS", [])
+            # so some fields related ExtraInfo are not editable in the standad implementation.
 
-            if extra_account_user_fields:
-                for field, value in request.data.items():
-                    if field in extra_account_user_fields and hasattr(request.user, field):
-                        setattr(request.user, field, value)
+            extra_account_user_fields = getattr(settings, "EXTRA_ACCOUNT_USER_FIELDS", [])
+            required_user_extra_info_fields = getattr(settings, 'REQUIRED_USER_EXTRA_INFO_FIELDS', [])
```

Comment:
you are turning on different features with the same setting

Reply:
That is not ok? or why it is better to have different settings, I mean they are also related to extra_info fields...